### DISCUSSION TITLE
How to set preflight to false when using tailwind through cdn?

### DISCUSSION THREAD

**Discussion by priyanshuhaldar007** [tailwindlabs/tailwindcss#12478] [Help] [answered]
I'm using tailwind through the CDN link and due to preflight resets, tailwind has reset the default features of HTML elements. Is there any way to set it to false for the core plugins like we do when using library using npm as shown below.
<img width="343" alt="Screenshot 2023-11-23 at 5 57 56 PM" src="https://github.com/tailwindlabs/tailwindcss/assets/52622977/524cc4a8-333c-42c0-ac38-a02e2775a018">

**Comment by wongjn** [comment, marked as answer]
Yes there is! As [per the documentation](https://tailwindcss.com/docs/installation/play-cdn):

> Edit the `tailwind.config` object to [customize your configuration](https://tailwindcss.com/docs/configuration) with your own design tokens.

Thus, in some JavaScript after the CDN script, do:

```js
tailwind.config = {
  corePlugins: {
    preflight: false,
  },
}
```
